app/src/components/job/hr/JobDeletionPopUp.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { softDelete } from "src/api/jobs";
+import DeleteJobSuccess from "./DeleteJobSuccess";
+
+function DeletePopUp({ trigger, setTrigger, jobId, isActive, jobName }) {
+  const [jobIsActive, setJobIsActive] = useState(false);
+  const [errors, setErrors] = useState([]);
+  const [displayPopup, setDisplayPopup] = useState(false);
+
+  const resetTrigger = (e) => {
+    setTrigger(false);
+  };
+
+  function refreshPage() {
+    window.location.reload(false);
+  }
+
+  const handleConfirm = async (e) => {
+    e.preventDefault();
+    const res = await softDelete(jobId, jobIsActive);
+    if (res.detail) {
+      const errorList = [];
+      if (res.detail.map) {
```

Comment:
.map is an arr function so doing it will not really check for anything except checking if the method exists --- which it usually does.

Reply:
Ok will remove in upcoming commit!